src/components/robot-item-form.tsx

```diff
@@ -0,0 +1,157 @@
+import { makeStyles, TextField, Button } from '@material-ui/core';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import React, { useState, useEffect } from 'react';
+import { successMsg } from '../util/alerts';
+import fakePlaces from '../mock/data/places';
```

Comment:
There used to be a places field in `BuildingMap` but I think it's deprecated and no longer contains anything. @Yadunund  Do you know where should the places be acquired from now?

Once we can query the list of places, we should use that instead of a mock.

Reply:
From our previous discussion with Yadu

The start_name and finish_name should contain valid waypoint names as listed in BuildingMap->levels->nav_graphs[i]->vertices[j]->name. However as there is no public mapping between the nav_graph index (i) and the fleet name, it is not possible to know which fleet of robots uses which nav graph from the BuildingMap msg.

A node on Places:
BuildingMap->levels->places is not currently populated. The use of places is kind of deprecated. The information of locations that a robot can travel to is stored as a GraphNode in BuildingMap->levels->nav_graphs[i]->vertices. This is because there are named waypoints that Robot A can travel to but not Robot B.

This is a workaround, until we figure it out a good way to get that information (Through a message, through a service, etc). But please correct me if I'm wrong.

Reply:
@koonpeng I asked Matias to use a file (`places.ts`) to get a list of named waypoints for each fleet as this is not explicitly conveyed in the BuildingMap msg. We can iterate on other methods in the future. Perhaps the same service serving robot icons can fulfill this requirement.